# HG changeset 9684 patch
From: kaf24@firebug.cl.cam.ac.uk
# Node ID e8ca9f2cb0d1d2f186bd2219830c0e40510b9adb
# Parent  614cb7ab15a860e95cd810e7ae34b952adad8db2
Subject: Remove Emacs variables settings from tail of our Linux files.
Signed-off-by: Keir Fraser <keir@xensource.com>
xen-unstable changeset:   10030:48c0f5489d44feb74d0982652072eb0cf0aed048
xen-unstable date:        Thu May 18 11:39:03 2006 +0100

Acked-By: Jan Beulich <jbeulich@novell.com>

Index: head-2006-05-19/arch/i386/kernel/fixup.c
===================================================================
--- head-2006-05-19.orig/arch/i386/kernel/fixup.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/arch/i386/kernel/fixup.c	2006-05-19 10:43:40.000000000 +0200
@@ -84,13 +84,3 @@ static int __init fixup_init(void)
 	return 0;
 }
 __initcall(fixup_init);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/arch/i386/kernel/pci-dma-xen.c
===================================================================
--- head-2006-05-19.orig/arch/i386/kernel/pci-dma-xen.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/arch/i386/kernel/pci-dma-xen.c	2006-05-19 10:43:40.000000000 +0200
@@ -332,13 +332,3 @@ dma_sync_single_for_device(struct device
 		swiotlb_sync_single_for_device(dev, dma_handle, size, direction);
 }
 EXPORT_SYMBOL(dma_sync_single_for_device);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/arch/i386/kernel/swiotlb.c
===================================================================
--- head-2006-05-19.orig/arch/i386/kernel/swiotlb.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/arch/i386/kernel/swiotlb.c	2006-05-19 10:43:40.000000000 +0200
@@ -662,13 +662,3 @@ EXPORT_SYMBOL(swiotlb_map_page);
 EXPORT_SYMBOL(swiotlb_unmap_page);
 EXPORT_SYMBOL(swiotlb_dma_mapping_error);
 EXPORT_SYMBOL(swiotlb_dma_supported);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/arch/i386/kernel/time-xen.c
===================================================================
--- head-2006-05-19.orig/arch/i386/kernel/time-xen.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/arch/i386/kernel/time-xen.c	2006-05-19 10:43:40.000000000 +0200
@@ -1085,13 +1085,3 @@ static int __init xen_sysctl_init(void)
 	return 0;
 }
 __initcall(xen_sysctl_init);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/arch/i386/mm/hypervisor.c
===================================================================
--- head-2006-05-19.orig/arch/i386/mm/hypervisor.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/arch/i386/mm/hypervisor.c	2006-05-19 10:43:40.000000000 +0200
@@ -412,13 +412,3 @@ int write_ldt_entry(void *ldt, int entry
 		mach_lp, (u64)entry_a | ((u64)entry_b<<32));
 }
 #endif
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/arch/i386/mm/ioremap-xen.c
===================================================================
--- head-2006-05-19.orig/arch/i386/mm/ioremap-xen.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/arch/i386/mm/ioremap-xen.c	2006-05-19 10:43:40.000000000 +0200
@@ -452,13 +452,3 @@ void __init bt_iounmap(void *addr, unsig
 }
 
 #endif /* __i386__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/arch/i386/mm/pgtable-xen.c
===================================================================
--- head-2006-05-19.orig/arch/i386/mm/pgtable-xen.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/arch/i386/mm/pgtable-xen.c	2006-05-19 10:43:40.000000000 +0200
@@ -640,13 +640,3 @@ void _arch_exit_mmap(struct mm_struct *m
 	    (atomic_read(&mm->mm_count) == 1))
 		mm_unpin(mm);
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/arch/x86_64/mm/init-xen.c
===================================================================
--- head-2006-05-19.orig/arch/x86_64/mm/init-xen.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/arch/x86_64/mm/init-xen.c	2006-05-19 10:43:40.000000000 +0200
@@ -1089,13 +1089,3 @@ int in_gate_area_no_task(unsigned long a
 {
 	return (addr >= VSYSCALL_START) && (addr < VSYSCALL_END);
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/balloon/balloon.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/balloon/balloon.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/balloon/balloon.c	2006-05-19 10:43:40.000000000 +0200
@@ -580,13 +580,3 @@ EXPORT_SYMBOL_GPL(balloon_alloc_empty_pa
 EXPORT_SYMBOL_GPL(balloon_dealloc_empty_page_range);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blkback/blkback.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/blkback/blkback.c	2006-05-12 09:24:55.000000000 +0200
+++ head-2006-05-19/drivers/xen/blkback/blkback.c	2006-05-19 10:43:40.000000000 +0200
@@ -577,13 +577,3 @@ static int __init blkif_init(void)
 module_init(blkif_init);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blkback/common.h
===================================================================
--- head-2006-05-19.orig/drivers/xen/blkback/common.h	2006-05-12 09:24:36.000000000 +0200
+++ head-2006-05-19/drivers/xen/blkback/common.h	2006-05-19 10:43:40.000000000 +0200
@@ -137,13 +137,3 @@ int blkif_schedule(void *arg);
 void update_blkif_status(blkif_t *blkif); 
 
 #endif /* __BLKIF__BACKEND__COMMON_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blkback/interface.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/blkback/interface.c	2006-05-12 09:24:36.000000000 +0200
+++ head-2006-05-19/drivers/xen/blkback/interface.c	2006-05-19 10:43:40.000000000 +0200
@@ -169,13 +169,3 @@ void __init blkif_interface_init(void)
 	blkif_cachep = kmem_cache_create("blkif_cache", sizeof(blkif_t), 
 					 0, 0, NULL, NULL);
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blkback/vbd.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/blkback/vbd.c	2006-05-19 10:36:56.000000000 +0200
+++ head-2006-05-19/drivers/xen/blkback/vbd.c	2006-05-19 10:43:40.000000000 +0200
@@ -117,13 +117,3 @@ int vbd_translate(struct phys_req *req, 
  out:
 	return rc;
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blkback/xenbus.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/blkback/xenbus.c	2006-05-12 09:24:36.000000000 +0200
+++ head-2006-05-19/drivers/xen/blkback/xenbus.c	2006-05-19 10:43:40.000000000 +0200
@@ -412,14 +412,3 @@ void blkif_xenbus_init(void)
 {
 	xenbus_register_backend(&blkback);
 }
-
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blkfront/blkfront.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/blkfront/blkfront.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/blkfront/blkfront.c	2006-05-19 10:43:40.000000000 +0200
@@ -807,13 +807,3 @@ static void xlblk_exit(void)
 module_exit(xlblk_exit);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blkfront/block.h
===================================================================
--- head-2006-05-19.orig/drivers/xen/blkfront/block.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/blkfront/block.h	2006-05-19 10:43:40.000000000 +0200
@@ -153,13 +153,3 @@ int xlvbd_add(blkif_sector_t capacity, i
 void xlvbd_del(struct blkfront_info *info);
 
 #endif /* __XEN_DRIVERS_BLOCK_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blkfront/vbd.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/blkfront/vbd.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/blkfront/vbd.c	2006-05-19 10:43:40.000000000 +0200
@@ -315,13 +315,3 @@ xlvbd_del(struct blkfront_info *info)
 	blk_cleanup_queue(info->rq);
 	info->rq = NULL;
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blktap/blktap.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/blktap/blktap.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/blktap/blktap.c	2006-05-19 10:43:40.000000000 +0200
@@ -898,13 +898,3 @@ static int __init blkif_init(void)
 }
 
 __initcall(blkif_init);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blktap/common.h
===================================================================
--- head-2006-05-19.orig/drivers/xen/blktap/common.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/blktap/common.h	2006-05-19 10:43:40.000000000 +0200
@@ -98,13 +98,3 @@ void blkif_xenbus_init(void);
 irqreturn_t blkif_be_int(int irq, void *dev_id, struct pt_regs *regs);
 
 #endif /* __BLKIF__BACKEND__COMMON_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blktap/interface.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/blktap/interface.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/blktap/interface.c	2006-05-19 10:43:40.000000000 +0200
@@ -134,13 +134,3 @@ void __init blkif_interface_init(void)
 	blkif_cachep = kmem_cache_create(
 		"blkif_cache", sizeof(blkif_t), 0, 0, NULL, NULL);
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/blktap/xenbus.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/blktap/xenbus.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/blktap/xenbus.c	2006-05-19 10:43:40.000000000 +0200
@@ -221,13 +221,3 @@ void blkif_xenbus_init(void)
 {
 	xenbus_register_backend(&blkback);
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/char/mem.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/char/mem.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/char/mem.c	2006-05-19 10:43:40.000000000 +0200
@@ -180,13 +180,3 @@ struct file_operations mem_fops = {
 	.mmap		= mmap_mem,
 	.open		= open_mem,
 };
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/console/console.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/console/console.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/console/console.c	2006-05-19 10:43:40.000000000 +0200
@@ -636,13 +636,3 @@ static int __init xencons_init(void)
 module_init(xencons_init);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/console/xencons_ring.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/console/xencons_ring.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/console/xencons_ring.c	2006-05-19 10:43:40.000000000 +0200
@@ -139,13 +139,3 @@ void xencons_resume(void)
 {
 	(void)xencons_ring_init();
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/core/evtchn.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/core/evtchn.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/core/evtchn.c	2006-05-19 10:43:40.000000000 +0200
@@ -857,13 +857,3 @@ void __init init_IRQ(void)
 		irq_desc[pirq_to_irq(i)].handler = &pirq_type;
 	}
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/core/gnttab.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/core/gnttab.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/core/gnttab.c	2006-05-19 10:43:40.000000000 +0200
@@ -452,13 +452,3 @@ gnttab_init(void)
 }
 
 core_initcall(gnttab_init);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/core/reboot.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/core/reboot.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/core/reboot.c	2006-05-19 10:43:40.000000000 +0200
@@ -369,13 +369,3 @@ static int __init setup_shutdown_event(v
 }
 
 subsys_initcall(setup_shutdown_event);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/core/skbuff.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/core/skbuff.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/core/skbuff.c	2006-05-19 10:43:40.000000000 +0200
@@ -132,13 +132,3 @@ static int __init skbuff_init(void)
 core_initcall(skbuff_init);
 
 EXPORT_SYMBOL(__dev_alloc_skb);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/core/smpboot.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/core/smpboot.c	2006-05-12 09:24:52.000000000 +0200
+++ head-2006-05-19/drivers/xen/core/smpboot.c	2006-05-19 10:43:40.000000000 +0200
@@ -607,13 +607,3 @@ int setup_profiling_timer(unsigned int m
 	return -EINVAL;
 }
 #endif
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/core/xen_proc.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/core/xen_proc.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/core/xen_proc.c	2006-05-19 10:43:40.000000000 +0200
@@ -17,13 +17,3 @@ void remove_xen_proc_entry(const char *n
 {
 	remove_proc_entry(name, xen_base);
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/evtchn/evtchn.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/evtchn/evtchn.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/evtchn/evtchn.c	2006-05-19 10:43:40.000000000 +0200
@@ -452,13 +452,3 @@ module_init(evtchn_init);
 module_exit(evtchn_cleanup);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/net_driver_util.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/net_driver_util.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/net_driver_util.c	2006-05-19 10:43:40.000000000 +0200
@@ -56,13 +56,3 @@ int xen_net_read_mac(struct xenbus_devic
 	return 0;
 }
 EXPORT_SYMBOL_GPL(xen_net_read_mac);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/netback/common.h
===================================================================
--- head-2006-05-19.orig/drivers/xen/netback/common.h	2006-05-12 09:24:36.000000000 +0200
+++ head-2006-05-19/drivers/xen/netback/common.h	2006-05-19 10:43:40.000000000 +0200
@@ -121,13 +121,3 @@ struct net_device_stats *netif_be_get_st
 irqreturn_t netif_be_int(int irq, void *dev_id, struct pt_regs *regs);
 
 #endif /* __NETIF__BACKEND__COMMON_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/netback/interface.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/netback/interface.c	2006-05-12 09:24:36.000000000 +0200
+++ head-2006-05-19/drivers/xen/netback/interface.c	2006-05-19 10:43:40.000000000 +0200
@@ -317,13 +317,3 @@ void netif_disconnect(netif_t *netif)
 		BUG();
 	}
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/netback/loopback.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/netback/loopback.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/netback/loopback.c	2006-05-19 10:43:40.000000000 +0200
@@ -242,13 +242,3 @@ static void __exit loopback_exit(void)
 module_exit(loopback_exit);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/netback/netback.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/netback/netback.c	2006-05-12 09:26:11.000000000 +0200
+++ head-2006-05-19/drivers/xen/netback/netback.c	2006-05-19 10:43:40.000000000 +0200
@@ -850,13 +850,3 @@ static int __init netback_init(void)
 module_init(netback_init);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/netback/xenbus.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/netback/xenbus.c	2006-05-12 09:24:36.000000000 +0200
+++ head-2006-05-19/drivers/xen/netback/xenbus.c	2006-05-19 10:43:40.000000000 +0200
@@ -353,14 +353,3 @@ void netif_xenbus_init(void)
 {
 	xenbus_register_backend(&netback);
 }
-
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/netfront/netfront.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/netfront/netfront.c	2006-05-12 09:26:11.000000000 +0200
+++ head-2006-05-19/drivers/xen/netfront/netfront.c	2006-05-19 10:43:40.000000000 +0200
@@ -1512,14 +1512,3 @@ static void xennet_proc_delif(struct net
 }
 
 #endif
-
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/privcmd/privcmd.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/privcmd/privcmd.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/privcmd/privcmd.c	2006-05-19 10:43:40.000000000 +0200
@@ -290,13 +290,3 @@ static int __init privcmd_init(void)
 }
 
 __initcall(privcmd_init);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/tpmback/common.h
===================================================================
--- head-2006-05-19.orig/drivers/xen/tpmback/common.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/tpmback/common.h	2006-05-19 10:43:40.000000000 +0200
@@ -79,13 +79,3 @@ extern int num_frontends;
 #define MMAP_VADDR(t,_req) ((t)->mmap_vstart + ((_req) * PAGE_SIZE))
 
 #endif /* __TPMIF__BACKEND__COMMON_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/tpmback/interface.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/tpmback/interface.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/tpmback/interface.c	2006-05-19 10:43:40.000000000 +0200
@@ -182,13 +182,3 @@ void __init tpmif_interface_exit(void)
 {
 	kmem_cache_destroy(tpmif_cachep);
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/tpmback/tpmback.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/tpmback/tpmback.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/tpmback/tpmback.c	2006-05-19 10:43:40.000000000 +0200
@@ -1048,13 +1048,3 @@ static void __exit tpmback_exit(void)
 module_exit(tpmback_exit);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/tpmback/xenbus.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/tpmback/xenbus.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/tpmback/xenbus.c	2006-05-19 10:43:40.000000000 +0200
@@ -317,13 +317,3 @@ void tpmif_xenbus_exit(void)
 {
 	xenbus_unregister_driver(&tpmback);
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/tpmfront/tpmfront.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/tpmfront/tpmfront.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/tpmfront/tpmfront.c	2006-05-19 10:43:40.000000000 +0200
@@ -719,13 +719,3 @@ tpmif_exit(void)
 module_exit(tpmif_exit);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/tpmfront/tpmfront.h
===================================================================
--- head-2006-05-19.orig/drivers/xen/tpmfront/tpmfront.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/tpmfront/tpmfront.h	2006-05-19 10:43:40.000000000 +0200
@@ -28,13 +28,3 @@ struct tx_buffer {
 };
 
 #endif
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/util.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/util.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/util.c	2006-05-19 10:43:41.000000000 +0200
@@ -68,13 +68,3 @@ void unlock_vm_area(struct vm_struct *ar
 	preempt_enable();
 }
 EXPORT_SYMBOL_GPL(unlock_vm_area);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/xenbus/xenbus_backend_client.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/xenbus/xenbus_backend_client.c	2006-05-24 14:20:28.000000000 +0200
+++ head-2006-05-19/drivers/xen/xenbus/xenbus_backend_client.c	2006-05-24 14:21:23.000000000 +0200
@@ -140,13 +140,3 @@ int xenbus_unmap_ring(struct xenbus_devi
 EXPORT_SYMBOL_GPL(xenbus_unmap_ring);
 
 MODULE_LICENSE("Dual BSD/GPL");
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/xenbus/xenbus_client.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/xenbus/xenbus_client.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/xenbus/xenbus_client.c	2006-05-19 10:43:41.000000000 +0200
@@ -271,14 +271,3 @@ XenbusState xenbus_read_driver_state(con
 	return result;
 }
 EXPORT_SYMBOL_GPL(xenbus_read_driver_state);
-
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/xenbus/xenbus_comms.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/xenbus/xenbus_comms.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/xenbus/xenbus_comms.c	2006-05-19 10:43:41.000000000 +0200
@@ -206,13 +206,3 @@ int xb_init_comms(void)
 
 	return 0;
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/xenbus/xenbus_comms.h
===================================================================
--- head-2006-05-19.orig/drivers/xen/xenbus/xenbus_comms.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/xenbus/xenbus_comms.h	2006-05-19 10:43:41.000000000 +0200
@@ -41,13 +41,3 @@ int xs_input_avail(void);
 extern wait_queue_head_t xb_waitq;
 
 #endif /* _XENBUS_COMMS_H */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/xenbus/xenbus_dev.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/xenbus/xenbus_dev.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/xenbus/xenbus_dev.c	2006-05-19 10:43:41.000000000 +0200
@@ -243,13 +243,3 @@ xenbus_dev_init(void)
 }
 
 __initcall(xenbus_dev_init);
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/xenbus/xenbus_probe.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/xenbus/xenbus_probe.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/xenbus/xenbus_probe.c	2006-05-19 10:43:41.000000000 +0200
@@ -1080,14 +1080,3 @@ static int __init wait_for_devices(void)
 }
 
 late_initcall(wait_for_devices);
-
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/drivers/xen/xenbus/xenbus_xs.c
===================================================================
--- head-2006-05-19.orig/drivers/xen/xenbus/xenbus_xs.c	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/drivers/xen/xenbus/xenbus_xs.c	2006-05-19 10:43:41.000000000 +0200
@@ -844,13 +844,3 @@ int xs_init(void)
 
 	return 0;
 }
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/asm-i386/mach-xen/asm/hypercall.h
===================================================================
--- head-2006-05-19.orig/include/asm-i386/mach-xen/asm/hypercall.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/asm-i386/mach-xen/asm/hypercall.h	2006-05-19 10:43:41.000000000 +0200
@@ -330,13 +330,3 @@ HYPERVISOR_nmi_op(
 }
 
 #endif /* __HYPERCALL_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/asm-x86_64/mach-xen/asm/hypercall.h
===================================================================
--- head-2006-05-19.orig/include/asm-x86_64/mach-xen/asm/hypercall.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/asm-x86_64/mach-xen/asm/hypercall.h	2006-05-19 10:43:41.000000000 +0200
@@ -331,13 +331,3 @@ HYPERVISOR_nmi_op(
 }
 
 #endif /* __HYPERCALL_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/balloon.h
===================================================================
--- head-2006-05-19.orig/include/xen/balloon.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/balloon.h	2006-05-19 10:43:41.000000000 +0200
@@ -61,13 +61,3 @@ extern spinlock_t balloon_lock;
 #define balloon_unlock(__flags) spin_unlock_irqrestore(&balloon_lock, __flags)
 
 #endif /* __ASM_BALLOON_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/driver_util.h
===================================================================
--- head-2006-05-19.orig/include/xen/driver_util.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/driver_util.h	2006-05-19 10:43:41.000000000 +0200
@@ -14,13 +14,3 @@ extern void lock_vm_area(struct vm_struc
 extern void unlock_vm_area(struct vm_struct *area);
 
 #endif /* __ASM_XEN_DRIVER_UTIL_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/evtchn.h
===================================================================
--- head-2006-05-19.orig/include/xen/evtchn.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/evtchn.h	2006-05-19 10:43:41.000000000 +0200
@@ -114,13 +114,3 @@ static inline void notify_remote_via_evt
 extern void notify_remote_via_irq(int irq);
 
 #endif /* __ASM_EVTCHN_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/foreign_page.h
===================================================================
--- head-2006-05-19.orig/include/xen/foreign_page.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/foreign_page.h	2006-05-19 10:43:41.000000000 +0200
@@ -28,13 +28,3 @@
 	( (void (*) (struct page *)) (page)->mapping )
 
 #endif /* __ASM_XEN_FOREIGN_PAGE_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/gnttab.h
===================================================================
--- head-2006-05-19.orig/include/xen/gnttab.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/gnttab.h	2006-05-19 10:43:41.000000000 +0200
@@ -114,13 +114,3 @@ int gnttab_suspend(void);
 int gnttab_resume(void);
 
 #endif /* __ASM_GNTTAB_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/net_driver_util.h
===================================================================
--- head-2006-05-19.orig/include/xen/net_driver_util.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/net_driver_util.h	2006-05-19 10:43:41.000000000 +0200
@@ -46,13 +46,3 @@ int xen_net_read_mac(struct xenbus_devic
 
 
 #endif /* _ASM_XEN_NET_DRIVER_UTIL_H */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/public/evtchn.h
===================================================================
--- head-2006-05-19.orig/include/xen/public/evtchn.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/public/evtchn.h	2006-05-19 10:43:41.000000000 +0200
@@ -89,13 +89,3 @@ struct ioctl_evtchn_notify {
 	_IOC(_IOC_NONE, 'E', 5, 0)
 
 #endif /* __LINUX_PUBLIC_EVTCHN_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/public/privcmd.h
===================================================================
--- head-2006-05-19.orig/include/xen/public/privcmd.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/public/privcmd.h	2006-05-19 10:43:41.000000000 +0200
@@ -82,13 +82,3 @@ typedef struct privcmd_blkmsg
 	_IOC(_IOC_NONE, 'P', 3, sizeof(privcmd_mmapbatch_t))
 
 #endif /* __LINUX_PUBLIC_PRIVCMD_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/xen_proc.h
===================================================================
--- head-2006-05-19.orig/include/xen/xen_proc.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/xen_proc.h	2006-05-19 10:43:41.000000000 +0200
@@ -11,13 +11,3 @@ extern void remove_xen_proc_entry(
 	const char *name);
 
 #endif /* __ASM_XEN_PROC_H__ */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
Index: head-2006-05-19/include/xen/xenbus.h
===================================================================
--- head-2006-05-19.orig/include/xen/xenbus.h	2006-05-24 14:19:43.000000000 +0200
+++ head-2006-05-19/include/xen/xenbus.h	2006-05-19 10:43:41.000000000 +0200
@@ -294,13 +294,3 @@ void xenbus_dev_fatal(struct xenbus_devi
 
 
 #endif /* _XEN_XENBUS_H */
-
-/*
- * Local variables:
- *  c-file-style: "linux"
- *  indent-tabs-mode: t
- *  c-indent-level: 8
- *  c-basic-offset: 8
- *  tab-width: 8
- * End:
- */
